From 2c095fae49351a6be86934658e20de0e9f985061 Mon Sep 17 00:00:00 2001
From: wkawka <wiktor.kawka@3mdeb.com>
Date: Fri, 16 Sep 2022 11:09:29 +0200
Subject: [PATCH] Makefile: disable deprecated warnigs as errors

Signed-off-by: wkawka <wiktor.kawka@3mdeb.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 7f38bd6b..e3615a11 100644
--- a/Makefile
+++ b/Makefile
@@ -124,7 +124,7 @@ endif
 # Provide default CC and CFLAGS for firmware builds; if you have any -D flags,
 # please add them after this point (e.g., -DVBOOT_DEBUG).
 DEBUG_FLAGS := $(if ${DEBUG},-g -Og,-g -Os)
-WERROR := -Werror
+WERROR := -Werror -Wno-deprecated-declarations
 FIRMWARE_FLAGS := -nostdinc -ffreestanding -fno-builtin -fno-stack-protector
 COMMON_FLAGS := -pipe ${WERROR} -Wall -Wstrict-prototypes -Wtype-limits \
 	-Wundef -Wmissing-prototypes -Wno-trigraphs -Wredundant-decls -Wshadow \
-- 
2.25.1

